src/main/java/gov/nih/nci/evs/api/controller/BaseController.java

```diff
@@ -15,18 +20,34 @@ public class BaseController {
   /** The Constant log. */
   private static final Logger logger = LoggerFactory.getLogger(BaseController.class);
 
+  /* The terminology utils */
+  @Autowired TerminologyUtils termUtils;
+
+  /** The elastic query service. */
+  @Autowired ElasticQueryService elasticQueryService;
```

Comment:
I don't think you're using this anywhere.  don't bother injecting if it isn't used.  Each sub-controller injects it themselves.